Exiled.Events/Patches/Events/Scp330/Eating.cs

```diff
@@ -0,0 +1,77 @@
+// -----------------------------------------------------------------------
+// <copyright file="Eating.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.Patches.Events.Scp330
```

Comment:
Do not make multiple files for the same method, just make a transpiler that implements both events.